include/deal.II/lac/petsc_vector.h

```diff
@@ -262,6 +262,11 @@ namespace PETScWrappers
        */
       explicit Vector(const IndexSet &local, const MPI_Comm &communicator);
 
+      /**
+       * Copy constructor.
+       */
+      Vector(const Vector &v) = default;
+
```

Comment:
I didn't really think about it before but this has to be a bug since we do not have a default copy operator. We should just define this for now with the default constructor followed by calling `operator=`.

Reply:
I tried with exactly that before, but surprisingly all PETSc tests with this change worked. Let us play safe and change to the other code I had before.